tests/plugins/easyimage/manual/pastefromword.md

```diff
@@ -1,7 +1,7 @@
 @bender-tags: 4.9.0, feature, 932
 @bender-ui: collapsed
 @bender-ckeditor-plugins: sourcearea, wysiwygarea, floatingspace, toolbar, easyimage, undo, pastefromword
-
+@bender-include: ./_helpers/tools.js,../_helpers/tools.js
```

Comment:
Won't it have more sense to have one file with tests tools?

Reply:
It makes sense. I tried to avoid it as those duplications of tools appeared during one of feature/bug-fixes merged to easyimage.
I'll prepare merge for that.